pandas_sphinx_theme/docs-sidebar.html

```diff
@@ -1,5 +1,10 @@
 
 <div class="d-none d-xl-block">
+  {%- if sidebars != None %}
```

Comment:
```suggestion
  {%- if sidebars %}
```

Is there a reason to not just check the `sidebars` is either none, empty or otherwise Falsy?